docs/reference/settings/monitoring-settings.asciidoc

```diff
@@ -66,6 +66,10 @@ option in `kibana.yml` to the same value.
 You can update this setting through the
 <<cluster-update-settings,Cluster Update Settings API>>.
 
+`xpack.monitoring.elasticsearch.collection.enabled`::
+
+Controls whether statistics about your {es} cluster should be collected. Defaults to `true`.
```

Comment:
You should also add:

```asciidoc
+
You can update this setting through the
<<cluster-update-settings,Cluster Update Settings API>>.
```

It may also be helpful to add details about this setting relative to `xpack.monitoring.collection.enabled`. Something like

> ... Defaults to `true`. This is different from `xpack.monitoring.collection.enabled`, which allows you to enable or disable all monitoring collection. However, this setting simply disables the collection of Elasticsearch data while still allowing other data (e.g., Kibana, Logstash, Beats, or APM Server monitoring data) to use through this cluster.